Assets/Scripts/LevelManager.cs

```diff
@@ -32,13 +40,22 @@ void Update()
 
     public void LevelLost()
     {
-        isGameOver = true;
         gameText.text = "GAME OVER!";
         gameText.gameObject.SetActive(true);
 
         //AudioSource.PlayClipAtPoint(gameOverSFX, Camera.main.transform.position);
+        GameObject player = GameObject.FindGameObjectWithTag("Player");
+        CharacterController controller = player.GetComponent<CharacterController>();
+        controller.enabled = false;
```

Comment:
Instead of doing this, use the strat that I did for the kiosk which is `PlayerMoveController.locked = true` and `PlayerLookController.locked = true`, and then remember to reset those to false when `RestartAtCheckpoint` is called